kolibri/core/assets/src/vue/textbox.vue

```diff
@@ -0,0 +1,82 @@
+<template>
+
+  <ui-textbox
+    @focus="$emit('focus')"
+    @input="updateText()"
+    v-model = "currentText"
+    :disabled="disabled"
+    :placeholder = "placeholder"
+    :label = "label"
+    :aria-label = "ariaLabel"
+    :autocomplete = "autocomplete"
+    :autofocus = "autofocus"
+    :required = "required"
+    :type = "type"
+    :invalid="invalid"
+    class="koli-textbox">
```

Comment:
Just following the format of `icon-button`, actually. Could be used for styling and might be helpful in the markup, but useless right now. I can get rid of it.